CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Fixed an issue where **validate** command failed on valid complex layout rules and triggers.
 * Fixed an issue where *validate* command failed with release notes files for assets modeling rules folder.
 * Added support for `skip` property in **pre-commit** hooks.
+* Added support of the `List` content-item to the **prepare-content** and **split** commands.
```

Comment:
please document changes to upload in the change log.

Reply:
I added a description of the `download`
Regarding the `upload`, it remains the same situation